net/pfSense-pkg-OpenBGPD/files/usr/local/pkg/openbgpd.inc

```diff
@@ -266,7 +266,7 @@ EOF;
 }
 
 function openbgpd_restart() {
-	if (is_openbgpd_running() == true) {
+	if ((is_openbgpd_running() == true) && (file_exists("/var/run/bgpd.sock"))) {
```

Comment:
The check for `/var/run/bgpd.sock` should be done inside `is_openbgpd_running()` so other places that call it will benefit of it

Reply:
For now is only one caller, the restart function.
If the service is runs without the socket, it may still be functional.
Suggested changes added.